modules/cij/extractors/fio_json_read.py

```diff
@@ -33,6 +33,7 @@ def extract_metrics(tcase: TestCase) -> List[dict]:
                 fname=os.path.basename(fpath),
                 job_id=n,
             )
+            ctx["evars"] = tcase.evars
```

Comment:
Nice addition! I think it makes sense to add an `evar` arg to `fio_json.make_context`, so that we ensure that future extractors also set this value

Reply:
Agree, however, I could not quite see how to add it within the scope of the function... can you do a fix for it?

Reply:
Yep!